cmd/archeio/app/buckets.go

```diff
@@ -27,10 +28,12 @@ import (
 //
 // blobs in the buckets should be stored at /containers/images/sha256:$hash
 func awsRegionToS3URL(region string) string {
-	// for now always return @ameukam's test bucket
 	switch region {
+	case "us-west-1", "us-west-2", "us-east-1", "us-east-2", "eu-central-1", "ap-southeast-1", "ap-northeast-1", "ap-south-1":
+		return fmt.Sprintf("https://prod-registry-k8s-io-%v.s3.%v.amazonaws.com", region, region)
 	default:
-		return "https://painfully-really-suddenly-many-raccoon-image-layers.s3.us-west-2.amazonaws.com"
+		// default to us-east-2 for stability purposes
+		return "https://prod-registry-k8s-io-us-east-2.s3.us-east-2.amazonaws.com"
```

Comment:
This could surprise a caller who expects the function to succeed for any valid AWS region.

I'd be tempted to have a single format string in the function that is always used on the happy parh, or fail the lookup if we don't recognize the region (and then, callers must check for an error).

In case the region isn't recognized:
We can have a different function eg `awsFallbackS3Url` that returns a constant value, or we can serve from elsewhere.